src/GraphQL/GraphQLExtensions.cs

```diff
@@ -443,9 +443,19 @@ public static IValue AstFromValue(this object value, ISchema schema, IGraphType
                 return new GuidValue(guid);
             }
 
-            if(serialized is short int16)
+            if (serialized is sbyte @sbyte)
             {
-                return new ShortValue(int16);
+                return new SByteValue(@sbyte);
+            }
+
+            if (serialized is byte @byte)
+            {
+                return new ByteValue(@byte);
+            }
+
+            if(serialized is short @short)
```

Comment:
@KnightVision90 I will be happy to merge this after this small fix:
```c#
if(serialized is short @short)
```
->
```c#
if (serialized is short @short)
```

Reply:
Oops, that should be fixed now.